### PR TITLE
division operation works.

### LINKED ISSUE #4: Division Operation
```python
# Divides two numbers
def div(x, y):
```

### PR DESCRIPTION
**Pull request by mukister**

### CODE REVIEW

**Comment by frinzekt** [comment]
closes #4

If you add this word "closes #[issue number]" it will automatically close the issue once the pull request is merged

File: README.md [hunk 1]
```diff
@@ -1,2 +1,8 @@
 # DemoBranchPullRequest
  Demo for creating branches and pull request with approval and code review
+
+## Instruction
+1. Assign yourself a specific Issue
+2. Create a branch from dev that will address that issuee
+3. Implement feature to address that issue
+4. Create a pull-request towards dev
```
Comment (hadeneh) [inline README.md:8]: Good Instructions.

File: operations.py [hunk 2]
```diff
@@ -16,4 +16,4 @@ def sub(x, y):
 # STUB
 # Divides two numbers
 def div(x, y):
-    return 0
\ No newline at end of file
+    return x/y
```
Comment (hadeneh) [inline operations.py]: Good Implementation! I am happy.

**Review by hadeneh — APPROVED** [review]
Tested and Completed.

**Pull request merged**